docs/devel/tree-info.rst

```diff
@@ -0,0 +1,17 @@
+devel/tree-info
+===============
+
+.. dfhack-tool::
+    :summary: Print a tree_info visualization of the tree at the cursor.
+    :tags: adventure fort inspection map plants
```

Comment:
```suggestion
    :tags: dev
```

Reply:
I'll register this script at https://docs.google.com/spreadsheets/d/1hiDlo8M_bB_1jE-5HRs2RrrA_VZ4cRu9VXaTctX_nwk/edit#gid=1774645373